content/en/blog/_posts/2020-08-14-introducing-hierarchical-namespaces.md

```diff
@@ -161,24 +161,22 @@ _also_ allow ingress traffic between all of those namespaces. The “tree” lab
 can only be applied by HNC, and is guaranteed to reflect the latest hierarchy.
 
 You can learn all about the features of HNC from the [user
-guide](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/hnc/docs/user-guide).
+guide](https://github.com/kubernetes-sigs/hierarchical-namespaces/tree/master/docs/user-guide).
 
 # Next steps and getting involved
 
 If you think that hierarchical namespaces can work for your organization, [HNC
-v0.5.1 is available on
-GitHub](https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.5.1).
+v1.0.0 is available on
+GitHub](https://github.com/kubernetes-sigs/hierarchical-namespaces/releases/tag/v1.0.0).
 We’d love to know what you think of it, what problems you’re using it to solve
-and what features you’d most like to see added. As with all early software, you
-should be cautious about using HNC in production environments, but the more
-feedback we get, the sooner we’ll be able to drive to HNC 1.0.
+and what features you’d most like to see added. 
```

Comment:
Leave the original text. It'd be OK to have a _new_ article about the v1.0 release but that should have come out roughly 12 months ago.

Reply:
Sure. I can ask Adrian (author) in order to draft the updated version